swo_rql/query_builder.py

```diff
@@ -0,0 +1,364 @@
+from swo_rql import constants
+
+
+def parse_kwargs(query_dict):
```

Comment:
I would ask you to answer the question: "What is the ideal interface you want to have for RQL builder?"
And based on that, you can have some intermidiate solution that doesn't contradict with the future :-)